### ISSUE TITLE
`const` in prop-types `lib/ReactPropTypesSecret.js`

### ISSUE THREAD

**Issue by mridgway** [facebook/react#9370]
**Do you want to request a *feature* or report a *bug*?**

Bug: prop-types@15.5.0 contains an un-transpiled `const` in `lib/ReactPropTypesSecret.js`.

Not sure where the code for this lives.

**Comment by acdlite** [comment]
Fixing now!

**Comment by STRML** [comment]
Hmm, perhaps we need a karma / PhantomJS@1 test to check for this kind of thing in the future?

**Comment by acdlite** [comment]
Should be fixed! Want to give it another try?

**Comment by mridgway** [comment]
```
~/Projects 
15:31 $ mkdir test2
✔ ~/Projects 
15:31 $ cd test2/
✔ ~/Projects/test2 
15:31 $ yarn add react
yarn add v0.21.3
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
success Saved 17 new dependencies.
├─ asap@2.0.5
├─ core-js@1.2.7
├─ encoding@0.1.12
├─ fbjs@0.8.12
├─ iconv-lite@0.4.15
├─ is-stream@1.1.0
├─ isomorphic-fetch@2.2.1
├─ js-tokens@3.0.1
├─ loose-envify@1.3.1
├─ node-fetch@1.6.3
├─ object-assign@4.1.1
├─ promise@7.1.1
├─ prop-types@15.5.0-alpha.0
├─ react@15.5.0
├─ setimmediate@1.0.5
├─ ua-parser-js@0.7.12
└─ whatwg-fetch@2.0.3
✨  Done in 2.23s.
✔ ~/Projects/test2 
15:32 $ cat 
node_modules/ package.json  yarn.lock     
✔ ~/Projects/test2 
15:32 $ cat 
node_modules/ package.json  yarn.lock     
✔ ~/Projects/test2 
15:32 $ cat node_modules/prop-types/
README.md          factory.js         lib/               
checkPropTypes.js  index.js           package.json       
✔ ~/Projects/test2 
15:32 $ cat node_modules/prop-types/lib/ReactPropTypesSecret.js 
/**
 * Copyright 2013-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 */

'use strict';

const ReactPropTypesSecret = 'SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED';

module.exports = ReactPropTypesSecret;
```

**Comment by mridgway** [comment]
It looks like react@15.5.0 has a dependency on prop-types@15.5.0-alpha.0. Is that correct?

**Comment by acdlite** [comment]
Ahhh I see the problem. Yep, that's it.

**Comment by acdlite** [comment]
Fixing now

**Comment by acdlite** [comment]
Released 15.5.1 https://unpkg.com/react@15.5.1/package.json

**Comment by gaearon** [comment]
The usual procedure for any long-awaited release 😅

**Comment by mridgway** [comment]
👍  fixed for me locally now.

**Comment by acdlite** [comment]
Awesome, sorry for the trouble. Thanks for the quick report!

**Comment by manuelbertelli** [comment]
Strange, `node_modules/react/lib/ReactPropTypesSecret.js` is actually with `var` but `node_modules/react/dist/react.js` still has `const`.

**Comment by acdlite** [comment]
@manuelbertelli Yep, should be fixed in 15.5.2
